app/main.js

```diff
@@ -1,109 +1,80 @@
 const fs = require("fs-extra");
 const settings = require("electron-settings");
+const path = require("path");
+const ipc = require("electron").ipcRenderer;
 
-let lastDone = "up"; // imporant var used around main.js
+let ready = false;
+let filelocation = "up";
+const debug = true;
```

Comment:
while this works for now, it's usually convention in the node world to use the `NODE_ENV` or `NODE_DEBUG` variables to toggle debug behavior